core/rawdb/database.go

```diff
@@ -244,58 +244,48 @@ func Open(o OpenOptions) (ethdb.Database, error) {
 // InspectDatabase traverses the entire database and checks the size
 // of all different categories of data.
 func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
-	var (
-		codeToFetch   ethrawdb.DatabaseStat
-		syncPerformed ethrawdb.DatabaseStat
-		syncProgress  ethrawdb.DatabaseStat
-		syncSegments  ethrawdb.DatabaseStat
-	)
+	type stat = ethrawdb.DatabaseStat
```

Comment:
nit: doesn't seem necessary to define this but not against it either

Reply:
It was just to make the struct-pointer literals cleaner a few lines below, and avoid repetitive `&ethrawdb.DatabaseStat{}`. I'm happy either way too.